tools/pot/openvino/tools/pot/api/samples/classification/classification_simple_api_sample.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2020-2022 Intel Corporation
```

Comment:
I think samples without references in POT documentation will not work. Do you plan to extend the documentation?

Reply:
Sure but I don't think we should do everything in one go. We will discuss this.